doc/dev-status.rst

```diff
@@ -22,6 +22,8 @@ Last updated: ``2017-12-14``
 +--------------------------------------+--------+--------+--------+--------+--------------+--------+--------+--------+----------+
 |`py-autocrypt`_                       |✔       |✔       |✔       |✔       |✘             |✘       |✘       |✘       |✔         |
 +--------------------------------------+--------+--------+--------+--------+--------------+--------+--------+--------+----------+
+|`pyac`_                               |✔       |✔       |✔       |✔       |✘             |✔       |✔       |✔       |✔         |
```

Comment:
IMHO the "setup message" tick should indicate things work in both directions with ASM messages (e.g. Delta.chat only says "started" even though delta can send ASMs to enigmail successfully already).  You could add a footnote like K-9 does for keygen or enigmail for header-inject that says it's untested wrt to ASM interactions with enigmail/k9/delta.

Reply:
This sounds like a reasonable amendment to me; but if the columns are unclear, the legend should also be improved to make it less ambiguous.

Reply:
I agree with incrementally changing things, the table and legend needs more qualitifcation.